core/src/main/java/com/novoda/accessibility/AccessibilityServices.java

```diff
@@ -2,11 +2,12 @@
 
 import android.accessibilityservice.AccessibilityServiceInfo;
 import android.content.Context;
-import android.support.annotation.VisibleForTesting;
```

Comment:
@mr-archano decided on doing the bare minimum:

- no new artifact (in hindsight was not necessary anyway - espresso-support already depends on this library (but does not expose any of its classes to its consumers)), just make the enum public
- return String for now, list/iterable in future if necessary
- removed the extra APIs to on this class, so we only expose `isSwitchAccessEnabled()` since `isSpokenFeedbackEnabled()` should cover the other two for now